OctaneVSPlugin/ViewModel/FieldViewModel.cs

```diff
@@ -104,8 +104,19 @@ public object Content
                         IsChanged = true;
                         break;
                     case "integer":
-                        _parentEntity.SetIntValue(Name, int.Parse(value.ToString()));
-                        IsChanged = true;
+                        try
+                        {
+                            _parentEntity.SetIntValue(Name, int.Parse(value.ToString()));
+                            IsChanged = true;
+                        }
+                        catch (Exception ex)
+                        {
+                            if (ex is FormatException || ex is OverflowException)
+                            {
+                                _parentEntity.SetValue(Name, "");
+                                IsChanged = true;
+                            }
+                        }
```

Comment:
this is bad, we should fix it next sprint, hope mada won't kill you

Reply:
@atothhpe this is horrible, we will try to look for another way to solve this